back-end/test/getShowsTrakt.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const axios = require('axios')
+const sinon = require('sinon')
+const expect = chai.expect
+const server = require('../app.js')
+const { mockErrorMessage, mockPopularShows } = require('../MockData.js')
+chai.use(chaiHttp)
+
+describe('GET /shows-trakt', () => {
+  let stub
+
+  afterEach(() => {
+    stub.restore()
+  })
+
+  it('should return 200 OK and data', async () => {
```

Comment:
I'd alter this description of the test to mention this is when there is no query